consensus/istanbul/backend/engine.go

```diff
@@ -103,14 +105,22 @@ func (sb *Backend) Author(header *types.Header) (common.Address, error) {
 // VerifyHeader checks whether a header conforms to the consensus rules of a
 // given engine. Verifies the seal regardless of given "seal" argument.
```

Comment:
This comment is now now confusing. It talks about 'the' seal but there are 2 seals, so it would be good to make it explicit what seals it checks.

Reply:
Since this is the same diff as the one applied in the release branches, I'd like for it to stay the same, and maybe we can update changes in a different PR?

Reply:
Ok, lets keep this the same.